client/ayon_max/plugins/publish/save_scenes_for_cameras.py

```diff
@@ -90,13 +93,10 @@ def process(self, instance):
                 for script in scripts:
                     tmp.write(script + "\n")
 
-            try:
-                current_filepath = current_filepath.replace("\\", "/")
-                tmp_script_path = tmp_script_path.replace("\\", "/")
-                run_subprocess([maxbatch_exe, tmp_script_path,
-                                "-sceneFile", current_filepath])
-            except RuntimeError:
-                self.log.debug("Checking the scene files existing")
+            current_filepath = current_filepath.replace("\\", "/")
+            tmp_script_path = tmp_script_path.replace("\\", "/")
+            run_subprocess([maxbatch_exe, tmp_script_path,
+                            "-sceneFile", current_filepath])
```

Comment:
Let's connect the the plug-in's logger.
```suggestion
                            "-sceneFile", current_filepath],
                            logger=self.log)
```